container-search/pom.xml

```diff
@@ -223,7 +223,7 @@
             <arg>-Xlint:-deprecation</arg>
             <arg>-Xlint:-dep-ann</arg>
             <arg>-Xlint:-cast</arg>
-            <arg>-Werror</arg>
+            <!--arg>-Werror</arg-->
```

Comment:
Did you intend to include this change in this PR?

Reply:
sorry, thanks for catching